### ISSUE TITLE
Проигрыш в случае быстрого нажатия

### ISSUE THREAD

**Issue by anonimizerme** [spider4216/snake#1]
Пример ситуации:
- змейка двигается влево
- быстро нажимаем вверх и вправо
- получаем сообщением о проигрыше

Ситуация возникает из-за того, что фактически змейка еще не совершила переход на клетку вверх, но направление уже установлено вверх, что позволяет ему измениться на движение вправо при быстро нажатии вправо. Соответственно змейка упирается сама в себя на той же линии

**Comment by spider4216** [comment]
Да, такая проблема наблюдается, будет исправлена при первой возможности. Вы также можете прислать исправления в виде пулл реквеста при желании
